config/checkstyle_checks.xml

```diff
@@ -350,7 +350,7 @@
             DetailsAST, CheckstyleException, UnsupportedEncodingException, BuildException, ConversionException, FileNotFoundException, TestException"/>
     </module>
     <module name="ClassFanOutComplexity">
-        <property name="max" value="25"/>
+        <property name="max" value="26"/>
```

Comment:
@timurt 
Please, create an issue. Explain, what your problem is there. Make a suppression, put a link to the issue into suppression file.